code/script.js

```diff
@@ -262,82 +322,146 @@ const selectQuestion = () => {
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
 
-  if (category === 'hair color') {
+const value = questions.value
+ if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
+      value: value,
       // 👆 add the value from the input here
       category: category,
     }
   } else if (category === 'eye color') {
     // Set this up your self
```

Comment:
I think these comments are there as a help for us in creating the project but they can be removed before publishing.